firmware/library/L3_HAL/Piezo_Driver.hpp

```diff
@@ -0,0 +1,39 @@
+// This driver is a layout for the piezo buzzer controller implementation
+// It contains the interface, which is inherited by the "Buzzer" class
+// which implements the functions in the interface
+// The constructor is responsible for initializing the port
+
+#pragma once
+
+// import only the necessary libraries to make this interface function
+#include <cstdint>
+
+// import the lpc_pwm class from SJOne library as our PWM driver
+#include "/Users/mianhashimshah/OpenSJ_Bluz/SJOne/L2_Drivers/lpc_pwm.hpp"
+
+// BuzzerInterface is inherited by Buzzer class
+class BuzzerInterface
+{
+ public:
+        // The init() class is responsible for selecting the pin configuration
+        virtual void init(uint8_t pinNumber) = 0;
```

Comment:
Spell this out fully and follow the google naming standard. 
See: https://google.github.io/styleguide/cppguide.html#Naming